plutus-ledger-api/src/PlutusLedgerApi/V1/Bytes.hs

```diff
@@ -28,12 +28,14 @@ import PlutusTx
 import PlutusTx.Prelude qualified as P
 import Prettyprinter.Extras (Pretty, PrettyShow (..))
 
+{- | An error that is encountered when converting a `Bytestring` to a `LedgerBytes`. -}
 data LedgerBytesError =
-    UnpairedDigit
-    | NotHexit Char
+    UnpairedDigit -- ^ Odd number of bytes.
+    | NotHexit Char -- ^ Not a hex digit.
```

Comment:
This will not be shown in haddock output because `LedgerBytesError` is not exported by this module. Which implies we don't have any (regression) tests for the `fromHex` function

Reply:
Good point. But let's leave this here anyway.